### PR TITLE
Use portable PathString() intead of literal.

### PR DESCRIPTION
**Pull request by ashari4**
**Description**: Describe your changes.

`ort_kernel_invoker` now uses the portable `PathString` constructor instead of a literal.

**Motivation and Context**
- Why is this change required? What problem does it solve?
 #8486

### CODE REVIEW

File: onnxruntime/core/eager/ort_kernel_invoker.cc [hunk 1]
```diff
@@ -21,7 +21,7 @@ common::Status ORTInvoker::Invoke(const std::string& op_name,
   Model model("test", 
               false, 
               ModelMetaData(),
-              "",
+              PathString(),
```
Comment (snnn) [inline onnxruntime/core/eager/ort_kernel_invoker.cc:24]: Or: ORT_TSTR("")

Comment (ashari4) [reply onnxruntime/core/eager/ort_kernel_invoker.cc]: Thanks - I've made the change.

**Review by snnn — APPROVED** [review]

**Pull request merged**